estate/models/estate_property.py

```diff
@@ -0,0 +1,20 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "estate property info"
```

Comment:
The description should usually be title-cased
```suggestion
    _description = "Estate Property Info"
```